src/Renderer/Camera.js

```diff
@@ -174,4 +174,30 @@ Camera.prototype.getFrustumLocalSpace = function (position, quaternion) {
 };
 
 
+Camera.prototype.getRTCMatrixFromCenter = function (center) {
```

Comment:
This function does not compute the RTC matrix for a node but for a position in space, relative to the current camera instance.

It could be more aptly named, like "computeRTCMatrixAtPoint".

I think Camera is a good place for this function, since other computing functions related to the camera (such as SSE) are in the same class.